app/src/main/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionScreen.kt

```diff
@@ -219,6 +219,14 @@ class TeleconsultSharePrescriptionScreen constructor(
     downloadButton.setButtonState(Enabled)
   }
 
+  override fun showShareProgress() {
```

Comment:
Just an FYI, we usually define these methods when implementing the UiRenderer.

Reply:
Noted.